src/Adapter/Cart/CartPresenter.php

```diff
@@ -232,20 +239,20 @@ public function present(Cart $cart)
         $subtotals['shipping'] = [
             'type' => 'shipping',
             'label' => $this->translator->trans('Shipping', [], 'Cart'),
-            'amount' => $shipping_cost != 0 ? $this->pricePresenter->convertAndFormat($shipping_cost) : $this->translator->trans('Free', [], 'Cart'),
+            'amount' => $shipping_cost != 0 ? $this->priceFormatter->convertAndFormat($shipping_cost) : $this->translator->trans('Free', [], 'Cart'),
         ];
 
         $subtotals['discounts'] = [
```

Comment:
Not related to this PR, but, if there are no discounts, shouldn't `$subtotals['discounts']` just be empty instead of containing a 0 discount?